Sources/Interaction/Style/InteractorStyleUnicam/example/index.js

```diff
@@ -0,0 +1,108 @@
+
+const fullScreenRenderWindow = vtkFullScreenRenderWindow.newInstance({
+  background: [0, 0, 0],
+});
+const renderWindow = fullScreenRenderWindow.getRenderWindow();
+const renderer = fullScreenRenderWindow.getRenderer();
+fullScreenRenderWindow.addController(controlPanel);
+
+// ----------------------------------------------------------------------------
+// Interactor style setup
+// ----------------------------------------------------------------------------
+
+const interactorStyle = vtkInteractorStyleUnicam.newInstance();
+interactorStyle.setWorldUpVec([0, 1, 0]);
+renderWindow.getInteractor().setInteractorStyle(interactorStyle);
+
+// ----------------------------------------------------------------------------
+// Basic actor setup
+// ----------------------------------------------------------------------------
+
+const coneSource = vtkConeSource.newInstance({ height: 1.0 });
+const mapper = vtkMapper.newInstance();
+mapper.setInputConnection(coneSource.getOutputPort());
+
+const actor = vtkActor.newInstance();
+actor.rotateZ(90);
+actor.setMapper(mapper);
+
+renderer.addActor(actor);
+renderer.resetCamera();
+renderWindow.render();
+
+// ----------------------------------------------------------------------------
+// Orientation widget setup
+// ----------------------------------------------------------------------------
+const axes = vtkAxesActor.newInstance({ pickable: false });
+const orientationWidget = vtkOrientationMarkerWidget.newInstance({
+  actor: axes,
+  interactor: renderWindow.getInteractor(),
+});
+orientationWidget.setViewportCorner(
+  vtkOrientationMarkerWidget.Corners.BOTTOM_RIGHT
+);
+orientationWidget.setMinPixelSize(100);
+orientationWidget.setMaxPixelSize(300);
+
+// ----------------------------------------------------------------------------
+// UI setup
+// ----------------------------------------------------------------------------
+
+function updateWorldUpVector() {
+  const useWorldUpVector = document.querySelector('.useWorldUpVector').checked;
+
+  let coordinates = Array.from(document.querySelector('.coordinates').children);
+  coordinates.forEach((element) => {
+    element.disabled = !useWorldUpVector;
+  });
+
+  if (useWorldUpVector) {
+    coordinates = coordinates.map((item) => {
+      return Number.parseFloat(item.value);
+    });
+    const validCoordinates = !coordinates.some((value) => {
+      return Number.isNaN(value);
+    });
+
+    if (validCoordinates) {
+      interactorStyle.setUseWorldUpVec(true);
+      interactorStyle.setWorldUpVec(...coordinates);
+    } else {
+      interactorStyle.setUseWorldUpVec(false);
+    }
+  } else {
+    interactorStyle.setUseWorldUpVec(false);
+  }
+}
+
+document.querySelector('.useWorldUpVector').oninput = () => {
+  updateWorldUpVector();
+  renderWindow.render();
+};
+document.querySelector('.coordinates').oninput = () => {
+  updateWorldUpVector();
+  renderWindow.render();
+};
+
```

Comment:
please add a checkbox that toggles from perspective to orthographic camera projection